controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/VersionStatusSerializer.java

```diff
@@ -107,12 +128,32 @@ private VespaVersion vespaVersionFromSlime(Inspector object) {
                                 Instant.ofEpochMilli(object.field(committedAtField).asLong()),
                                 object.field(isControllerVersionField).asBool(),
                                 object.field(isSystemVersionField).asBool(),
-                                object.field(isReleasedField).valid() ? object.field(isReleasedField).asBool() : true,
-                                configServersFromSlime(object.field(configServersField)),
+                                !object.field(isReleasedField).valid() || object.field(isReleasedField).asBool(),
+                                nodeVersionsFromSlime(object),
                                 VespaVersion.Confidence.valueOf(object.field(confidenceField).asString())
         );
     }
 
+    private NodeVersions nodeVersionsFromSlime(Inspector root) {
+        var nodeVersions = new LinkedHashMap<HostName, NodeVersion>();
+        var nodeVersionsRoot = root.field(nodeVersionsField);
+        if (nodeVersionsRoot.valid()) {
+            nodeVersionsRoot.traverse((ArrayTraverser) (i, entry) -> {
+                var hostname = HostName.from(entry.field(hostnameField).asString());
+                var version = Version.fromString(entry.field(versionField).asString());
```

Comment:
Shouldn't be necessary to store this, as it's implicit by under which `VespaVersion` the `NodeVersion` is stored.